src/Main.elm

```diff
@@ -36,65 +37,68 @@ type alias Model =
     }
 
 
+startGame : Difficulty -> Model
+startGame difficulty =
+    { initialModel
+        | game = difficulty
+        , grid = Grid.fromDimensions (getDimensions difficulty)
+    }
+
+
+initialModel : Model
+initialModel =
+    { grid = Grid.fromDimensions (getDimensions initialDifficulty)
+    , activeCell = Nothing
+    , pressingFace = False
+    , game = initialDifficulty
+    , time = 0
+    , mode = Start
+    , isRightClicked = False
+    , menu = Nothing
+    }
+
+
 type Difficulty
-    = Beginner
-    | Intermediate
-    | Expert
```

Comment:
why take these out?

Reply:
I wish to tackle all the settings pertaining to beginner / intermediate / expert in just one place. right now they are defined in two places: the beginner's grid, and beginner's bomb count both need to be accounted for.

Reply:
Please review in the latest commits for the comment I will close this review.
rgds
Ben